pkg/cloudprovider/providers/gce/gce_loadbalancer_external.go

```diff
@@ -500,6 +510,34 @@ func (gce *GCECloud) ensureTargetPoolAndHealthCheck(tpExists, tpNeedsRecreation
 			glog.Infof("ensureTargetPoolAndHealthCheck(%s): Updated target pool (with %d hosts).", lbRefStr, len(hosts)-maxTargetPoolCreateInstances)
 		}
 	} else if tpExists {
+
```

Comment:
Add a general comment regarding what is being done here.

Reply:
Comment added. Not sure whether the comment is appropriate.